docs/resource-sharing/os-backfill-containers.md

```diff
@@ -213,6 +213,26 @@ the `--memory` argument tells Docker to kill the container if its total memory u
     Additionally, the processes that manage jobs also use some amount of memory.
     Therefore, it is important to give the container some extra room.
 
+
+### Advanced: Advertising additional pilot attributes
+
+You can put arbitrary additional attributes in the machine ads that you advertise to the OSPool.
```

Comment:
```suggestion
You can put arbitrary additional attributes in the machine ads that the glidein advertises to the OSPool.
```

Reply:
We don't call them glideins unless they're submitted by a factory but I can use "the pilot advertises" instead